backend/src/controllers/entry/entry.ts

```diff
@@ -180,7 +190,7 @@ export const deleteEntry = async (req, res, next) => {
 
     // Commit the transaction
     await session.commitTransaction();
-  } catch (error) {
```

Comment:
There's a lot of these unused errors throughout the code. It's alright that you removed them, but I was thinking it might be useful to use them when we implement more robust server side logging. It'd be nice to see these errors and the stack trace written to a file at some point. See Issue #184

Reply:
Sure we could always add them back in later. But it might be possible to have the logging happen without adding a log statement here too. It depends on the logging solution we settle on.